CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 * Added functions to retrieve and use VCD version `client.GetVcdVersion`, `client.GetVcdShortVersion`, `client.GetVcdFullVersion`, `client.VersionEqualOrGreater` [#339](https://github.com/vmware/go-vcloud-director/pull/339)
 * Added methods `VM.UpdateStorageProfile`, `VM.UpdateStorageProfileAsync` [#338](https://github.com/vmware/go-vcloud-director/pull/338)
 * Added methods `adminVdc.UpdateStorageProfile` [#340](https://github.com/vmware/go-vcloud-director/pull/340)
+* Added support for bearer tokens [#341](https://github.com/vmware/go-vcloud-director/pull/341)
```

Comment:
Would it be worth mentioning that the selection between old and new token types are done automatically? I like that in this PR :)

Reply:
added "transparent" to signify that the token just works.